microsite/sidebars.json

```diff
@@ -149,6 +149,14 @@
           "integrations/bitbucket/discovery"
         ]
       },
+      {
+        "type": "subcategory",
+        "label": "Bitbucket Cloud",
+        "ids": [
+          "integrations/bitbucketCloud/locations",
+          "integrations/bitbucketCloud/discovery"
+        ]
```

Comment:
Could you add this to the root `mkdocs.yml` as well? It's so that we can have TechDocs rendered too :grin: